### ISSUE TITLE
UI for control

### ISSUE THREAD

**Issue by brashrebel** [brashrebel/wptv#4]
Not sure whether this is a good idea or not but am contemplating adding admin control(s) for the plugin. Some site admins may want them disabled....or possibly other things.

**Comment by doctored** [comment]
sounds like its worth a shot at least at this phase i know some people don't like options in general though :/

**Comment by brashrebel** [comment]
You're right @doctored. It is my belief that as a standalone plugin that users are consciously installing that adding an option or two for tweaking what, when and where videos are displayed could be beneficial. However, if this were moved forward as a proposed feature plugin then new options would not likely be desirable.

I think I'll incorporate some for now for discussion purposes mostly.
